cmd/linuxptp-daemon/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fedora:32 AS builder
```

Comment:
We now have some Dockerfiles cmd/.., and some in root dir
Maybe ./Dockerfile.ptp-daemon  ?